packages/react/src/transactions/useCreatePost.ts

```diff
@@ -79,6 +81,27 @@ export type CreatePostBaseArgs = {
    * These are not the same as #hashtag in the publication content. Use these if you don't want to clutter the publication content with tags.
    */
   tags?: string[];
+  /**
+   * A list of attributes for the collect NFT.
+   *
+   * This is the NFT description visible on marketplaces like OpenSea.
+   */
+  attributes?: MetadataAttribute[];
+  /**
+   * The collect NFT image.
+   *
+   * This is the NFT image visible on marketplaces like OpenSea.
+   *
+   * DO NOT use this as primary storage for publication media. Use the `media` property instead.
+   * In the case your publication has many media consider to use this field as a static representation
+   * of the collect NFT. For example if the publication is an album of audio files this could well be
```

Comment:
nit: same again here with 'well', not necessarily needed 
```suggestion
   * of the collect NFT. For example if the publication is an album of audio files this could well be
```